src/ldp/http/BasicResponseWriter.ts

```diff
@@ -22,7 +22,7 @@ export class BasicResponseWriter extends ResponseWriter {
   public async canHandle(input: { response: HttpResponse; result: ResponseDescription | Error }): Promise<void> {
     if (input.result instanceof Error || input.result.metadata?.contentType === INTERNAL_QUADS) {
       this.logger.warn('This writer only supports binary ResponseDescriptions');
-      throw new UnsupportedHttpError('Only successful binary responses are supported');
+      throw new BadRequestHttpError('Only successful binary responses are supported');
```

Comment:
This one, for instance. We need a better sweep.

Reply:
I guess it pretty much depends on if this class takes as input something from the user side or something from the store side. I do think this writer is one of the few where that is the case (we have a `NotImplementedHttpError` for 501 btw).

Reply:
That does sort of make it awkward that we would have to decide when writing a handler where it would be in the path.

Reply:
I don't think so; this is a clear `NotImplementedError` for me.

Reply:
I was writing up a hypothetical example of where it could be confusing, but it's probably too specific and I'm overthinking it so let's just change this one (and the other response writer).